src/common/identity/selectors.js

```diff
@@ -476,3 +478,11 @@ export const selectCorporateProfile = createSelector(
 		members
 	})
 );
+
+export const selectPositionsForCompanyType = (state, companyType) => {
```

Comment:
please use `(state, props) => ..` pattern for new selectors

user `rootSelector` and `propsSelector` helpers to make life easier


So this should be:

```
export const selectPositionsForCompanyType = createSelector(
    state => selectIdAttributeTypeByUrl({attributeTypeUrl: CORPORATE_STRUCTURE}),
    propsSelector('companyType'),
    (attrType, props) => new CorporateStructureSchema(attrType.content).getPositionsForCompanyType(props.companyType)
);
```

to use: 

```
selectPositionsForCompanyType(state, {companyType: 'test'});
```